client/web/src/integration/context.ts

```diff
@@ -70,6 +61,7 @@ export const createWebIntegrationTestContext = async ({
     customContext = {},
 }: IntegrationTestOptions): Promise<WebIntegrationTestContext> => {
     const config = getConfig('disableAppAssetsMocking')
+    const bundles = getManifestBundles()
```

Comment:
Nit: to remove `const environment = bundles.environment` below

```suggestion
    const { environment, ...bundles } = getManifestBundles()
```